examples/dynamo/eg3/solver_mod.x90

```diff
@@ -437,7 +437,7 @@ contains
   call invoke_set_field_scalar(1.0_r_def, Ax)
   call invoke( matrix_vector_kernel_mm_type(lumped_weight,Ax,mm) )
 
-  !PSY call invoke ( divide_field(rhs, lumped_weight, lhs))
+  !PSY call invoke ( divide_fields(rhs, lumped_weight, lhs))
```

Comment:
Adapting eg3/solver_mod.x90 from PSy-Kal lite to PSyclone built-ins invokes and it is a much bigger task than it seems. There are a several issues, some of them:
-	Failure to parse expressions in calls, e.g.  –alpha, 1.0_r_def/alpha. I can work around this by defining new constants.
-	Complaints about passing arguments into kernels more than once. This is what I added new built-ins for, so I can replace them.
-	Failure to parse calls with input arguments that have loop indices in them, e.g. call invoke_inner_prod( v(k), w, h(k,j). I don’t think there is a way around that in this release of PSyclone.
There will probably be some more issues and I haven’t even started the proper grouping of invokes. 

I think re-arranging this solver example  from  PSy-Kal lite to PSyclone calls brings up enough work and changes to open a separate issue for that. 
I saved all the changes I made in another file so I can start working from them in a new issue.